src/cordova/platform.js

```diff
@@ -1,769 +0,0 @@
-/**
```

Comment:
Sure do! Basically, each module should have unit tests. A good rule of thumb is to have one spec file per module file.

To really get a sense of what coverage exists and what is missing, run `npm run cover` and get the code coverage report. Note that as of right now, the unit tests still have _some_ integration tests - so the code coverage report from a unit test standpoint is somewhat inaccurate (integration or end to end tests are very coarse, so they probably cover a lot of code unintentionally). I would recommend a) looking through the spec-cordova and spec-plugman tests for heavy i/o, very broad tests that we can classify as integration tests and moving them into integration-tests and b) replacing them with finer-grained unit tests.

This cordova-ios PR is an example of that general pattern: https://github.com/apache/cordova-ios/pull/319